dbcon/mysql/ha_mcs_ddl.cpp

```diff
@@ -107,6 +107,31 @@ inline uint32_t tid2sid(const uint32_t tid)
     return CalpontSystemCatalog::idb_tid2sid(tid);
 }
 
+static void decode_objectname(char *buf, const char *path, size_t buf_size)
+{
+    size_t new_path_len = filename_to_tablename(path, buf, buf_size);
+    buf[new_path_len] = '\0';
+}
+
+static void decode_file_path(const char *path, char *decoded_dbname,
+    char *decoded_tbname)
+{
+  // The format cont ains './' in the beginning of a path.
+  char *dbname_start = (char*) path + 2;
+  char *dbname_end = dbname_start;
+  while (*dbname_end != '/')
+    dbname_end++;
+
+  int cnt = dbname_end - dbname_start;
+  char *dbname = (char *)my_alloca(cnt + 1);
+  memcpy(dbname, dbname_start, cnt);
+  dbname[cnt] = '\0';
+  decode_objectname(decoded_dbname, dbname, FN_REFLEN);
+  my_afree(dbname);
+
+  char *tbname_start = dbname_end + 1;
+  decode_objectname(decoded_tbname, tbname_start, FN_REFLEN);
+}
```

Comment:
Indentation issue. `decode_file_path` function block is indented with 2 spaces, while `decode_objectname` is indented with 4 spaces.